### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by cherelemma**
In this project work, I and my partner

- Add a contact form and footer navigation section.
- Add a CSS for the contact form and footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,205 +17,228 @@
...
-
-        <hxr class="skils-title">
-          <img src="images/circle.png" alt="Diomand shape">
-          <h3 class="language">Skills</h3>
-        </hxr>
+    <footer> 
+      <section class="contact">
+        <p>I'm always interested in hearing about new projects,
+        so if you'd like to chat please get in touch.</p>
+        <form action="https://formspree.io/f/{form_id}" method="post">
```
Comment (christianbiring1) [inline index.html:224]: - [ ] Kindly make sure you use the form spree service correctly. You can check the tutorial [here](https://formspree.io/html/) on the provided lesson or  you can see this video [here](https://www.youtube.com/watch?v=H_MUJ--iAcc&t=6s)

File: index.html [hunk 2]
```diff
@@ -17,205 +17,228 @@
...
+          <input type="text" id="fullname" placeholder="Full Name" maxlength="30" required>
+          <input type="email" id="email" placeholder="Email address" required>
+          <textarea id="message" maxlength="50" placeholder="Enter text here" required></textarea>
```
Comment (christianbiring1) [inline index.html:227]: - [ ] Kindly make sure you use the correct `length` on textarea. It's supposed to be `500` and not `50`

File: index.html [hunk 3]
```diff
@@ -17,205 +17,228 @@
...
+    
+      <section class="about-us">
+        <h2 class="primary-text">About me</h2>
+        <p class="description">Hello I’m a software developer! I can help you build a product, 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
+          <button type="button">Get My Resume</button>
+      </section>   
+
```
Comment (christianbiring1) [inline index.html:181]: - [ ] Kinldy make sure you use the correct typo (`font-familly, size`) as the provided design 
Your design :point_down: 
image

Figma design :point_down: 
image

File: index.html [hunk 4]
```diff
@@ -17,205 +17,228 @@
...
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
+            <li><a href="https://github.com/cherelemma/"><img src="images/frame1.png" alt="Github Icon"></a></li>
+            <li><a href="https://www.linkedin.com/in/chere-lemma27211613/"><img src="images/frame2.png" alt="Linkedin Icon"></a></li>
+            <li><a href="#"><img src="images/frame3.png" alt="Angel list Icon"></a></li>
+            <li><a href="https://twitter.com/Chere21271613"><img src="images/frame4.png" alt="Twitter Icon"></a></li>
+            <li><a href="#"><img src="images/frame5.png" alt="Medium Icon"></a></li>
           </ul>
```
Comment (christianbiring1) [inline index.html:239]: - [ ] Kindly consider using the correct path for your social icon as you can see how they are displayed on your page

Your design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @cherelemma @Eugeneiyukhun , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅
## Hi Team👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**



## Highlights:
* Well written PR description ✔️ 
* All tests passed ✔️ 
* Well written commit messages ✔️ 

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**